picatrix/lib/manager.py

```diff
@@ -104,6 +106,7 @@ def get_magic_info(cls, as_pandas: bool = False) -> Union[
           'function': f'{magic_name}_func',
           'description': description}
       entries.append(magic_dict)
+
     df = pandas.DataFrame(entries)
     return df[
         ['name', 'description', 'line', 'cell', 'function']].sort_values('name')
```

Comment:
Consider adding a constant MAGICS_DF_COLUMNS = ['name', 'description', 'line', 'cell', 'function'] so you can use it in tests.

Reply:
ack